content/departments/product-engineering/engineering/cloud/delivery/managed/enable_executors_process.md

```diff
@@ -0,0 +1,87 @@
+# Enabling executors
+
+Enabling executor on a managed instances is initiated by "?". There are a few logical steps to follow.
+
+1. Update the managed instances terraform to include executor modules
+1. Setup billing alert for the executor resources
+
+## Update the managed instances terraform
+
+The change to terraform will be done by Batch Change team. Batch Change teamm will ping us the Pull Request for the changes and Delivery team will apply the change.
+
+## Setup billing alert
+
+As of 2022-03-08, we give customers $500 compute credits for executors usage, so we need to setup billing alert.
```

Comment:
I'm not sure this number should be included in the handbook. It may not be consistent across plans.